src/Illuminate/Validation/Concerns/FormatsMessages.php

```diff
@@ -295,6 +296,53 @@ protected function replaceAttributePlaceholder($message, $value)
         );
     }
 
+    /**
+     * Replace the array attribute :key{n}/:index{n}/:iteration{n} placeholders in the given message.
+     *
+     * @param  string  $message
+     * @param  string  $attribute
+     * @return string
+     */
+    protected function replaceArrayPlaceholders($message, $attribute)
+    {
+        $explicitKeys = $this->getExplicitKeys($attribute);
```

Comment:
I'm not sure if this line is a breaking change and I should check for method existence or make a pull request for the 9.x branch instead.